tests/measure.py

```diff
@@ -347,8 +347,8 @@ def testPsfFlux(self):
         schema.addField("centroid_flag", type='Flag')
         sfm_config = measBase.SingleFrameMeasurementConfig()
         sfm_config.doReplaceWithNoise = False
-        sfm_config.plugins = ["base_CircularApertureFlux", "base_PsfFlux"]
-        sfm_config.slots.centroid = None
+        sfm_config.plugins = ["base_GaussianCentroid", "base_CircularApertureFlux", "base_PsfFlux"]
+        sfm_config.slots.centroid = "base_GaussianCentroid"
```

Comment:
In fact, this test is already defining and using a centroid slot. Isn't it simpler to just use that rather than also running the Gaussian centroid algorithm? I think something like this would work:

```python
--- a/tests/measure.py
+++ b/tests/measure.py
@@ -347,8 +347,8 @@ class GaussianPsfTestCase(unittest.TestCase):
         schema.addField("centroid_flag", type='Flag')
         sfm_config = measBase.SingleFrameMeasurementConfig()
         sfm_config.doReplaceWithNoise = False
-        sfm_config.plugins = ["base_GaussianCentroid", "base_CircularApertureFlux", "base_PsfFlux"]
-        sfm_config.slots.centroid = "base_GaussianCentroid"
+        sfm_config.plugins = ["base_CircularApertureFlux", "base_PsfFlux"]
+        sfm_config.slots.centroid = "centroid"
         sfm_config.slots.shape = None
         sfm_config.slots.psfFlux = None
         sfm_config.slots.instFlux = None
@@ -359,7 +359,6 @@ class GaussianPsfTestCase(unittest.TestCase):
         sfm_config.plugins["base_CircularApertureFlux"].radii = [rad]
         task = measBase.SingleFrameMeasurementTask(schema, config=sfm_config)
         measCat = afwTable.SourceCatalog(schema)
-        measCat.defineCentroid("centroid")
         source = measCat.addNew()
         source.set("centroid_x", self.xc)
         source.set("centroid_y", self.yc)
```

Alternatively, just use the slot defined by the algorithm:

```python
--- a/tests/measure.py
+++ b/tests/measure.py
@@ -342,9 +342,6 @@ class GaussianPsfTestCase(unittest.TestCase):
         rad = 10.0

         schema = afwTable.SourceTable.makeMinimalSchema()
-        schema.addField("centroid_x", type=float)
-        schema.addField("centroid_y", type=float)
-        schema.addField("centroid_flag", type='Flag')
         sfm_config = measBase.SingleFrameMeasurementConfig()
         sfm_config.doReplaceWithNoise = False
         sfm_config.plugins = ["base_GaussianCentroid", "base_CircularApertureFlux", "base_PsfFlux"]
@@ -359,10 +356,9 @@ class GaussianPsfTestCase(unittest.TestCase):
         sfm_config.plugins["base_CircularApertureFlux"].radii = [rad]
         task = measBase.SingleFrameMeasurementTask(schema, config=sfm_config)
         measCat = afwTable.SourceCatalog(schema)
-        measCat.defineCentroid("centroid")
         source = measCat.addNew()
-        source.set("centroid_x", self.xc)
-        source.set("centroid_y", self.yc)
+        source.set("base_GaussianCentroid_x", self.xc)
+        source.set("base_GaussianCentroid_y", self.yc)
         task.run(measCat, self.exp)
         for algName in ["base_CircularApertureFlux_10_0", "base_PsfFlux"]:
             flux = source.get(algName + "_flux")
```